core/blockchain.go

```diff
@@ -238,7 +248,19 @@ func ExecuteBlockEphemerallyForBSC(
 		return nil, fmt.Errorf("writing changesets for block %d failed: %w", header.Number.Uint64(), err)
 	}
 
-	return receipts, nil
+	execRs := &EphemeralExecResult{
+		TxRoot:      types.DeriveSha(includedTxs),
+		ReceiptRoot: receiptSha,
+		Bloom:       bloom,
+		// LogsHash:          rlpHash(blockLogs),
```

Comment:
why is LogsHash not present?

Reply:
because EBEforBSC didn't have anything related to it. So, I thought maybe it's kinda useless. But, I add it just in case